### DISCUSSION TITLE
Which U.S. president was a fashion model in his youth?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#111] [Galaxy Trivia] [answered]
**Hint:** 38th

**Comment by Tynab** [comment, marked as answer]
The U.S. president who was a fashion model in his youth is Gerald Ford. Before he entered politics, Ford worked as a model and appeared on the cover of Cosmopolitan magazine in 1942. He was also featured in advertisements for companies such as Colgate toothpaste and Florsheim shoes.
